features/step_definitions/view_dishes_steps.rb

```diff
@@ -0,0 +1,14 @@
+
+Given(/^there is a dish called "([^"]*)" on "([^"]*)"$/) do |dish, menu|
+  menu = Menu.create(name: menu)
```

Comment:
Ack, sorry guys but here `menu` and `menu` are two different things, so you should change the name of one. Ruby allows this but other languages would not. It's bad practice.